packages/medusa/src/services/order-edit.ts

```diff
@@ -382,4 +383,40 @@ export default class OrderEditService extends TransactionBaseService {
 
     return orderEdit
   }
+
+  async cancel(orderEditId: string, userId: string): Promise<OrderEdit> {
```

Comment:
todo: (sorry for not spotting in the first iteration) It should technically be fine to cancel an order without it being a user who does it (e.g., in a subscriber)

```suggestion
  async cancel(orderEditId: string, userId?: string): Promise<OrderEdit> {
```

Reply:
also maybe we should align on either `userId` like here or `{ loggedInUser }` as seen elsewhere; have no opinion